docs/installation/basic.md

```diff
@@ -15,28 +23,98 @@ Make sure you have the dependencies mentioned above installed before proceeding
 
 Run the commands mentioned below with the terminal active in the project's root directory.
 
+* **Step 0** - Clone the Open Event Server repository (from the development branch) and ```cd ``` into the directory.
+```sh
+git clone -b development https://github.com/fossasia/open-event-server.git
+cd open-event-server
+```
+**Note :** If you want to contribute, first fork the original repository and clone the forked repository into your local machine followed by ```cd``` into the directory
+```sh
+git clone https://github.com/USERNAME/open-event-server.git
+cd open-event-server
+```
+
+* **Step 1** - Install python3 requirements. You need to be present in the root directory of the project.
+
+# System Wide Installation
+
+```sh
+sudo -H pip3 install -r requirements.txt
+```
+hint: You may need to upgrade your pip version and install following packages if you encounter errors while installing the requirements.
+
+**Note:** For Mac OS Sierra users, if you get an error that 'openssl/aes.h' could not be found when installing requirements.txt using pip, try the steps shown here - [OSX openssl header error](https://tutorials.technology/solved_errors/1-OSX-openssl_opensslv_h-file-not-found.html)
+
+# Installation in Virtual Environment
```

Comment:
Move this above system wide installation and say that this is recommended over systemwide installation

Reply:
Done